src/api/searchApi.ts

```diff
@@ -235,6 +235,41 @@ export const fetchNviCandidates = async (results: number, from: number, query =
   return getNviCandidates.data;
 };
 
+enum NviCandidatesSearchParam {
+  Aggregation = 'aggregation',
```

Comment:
Bakend bruker `aggregationType` for dette akkurat nå, men har bestilt endring av det som vil komme ganske så snart, så bruker dette med en gang. Men inntil backend har rullet tut renaming vil vi få alle aggregations i stedet for bare den ene, som er helt OK